modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/action/EditEntryMVCActionCommand.java

```diff
@@ -536,14 +537,16 @@ private BlogsEntry _updateEntry(ActionRequest actionRequest)
 		ServiceContext serviceContext = ServiceContextFactory.getInstance(
 			BlogsEntry.class.getName(), actionRequest);
 
+		if (_featureFlagManager.isEnabled("LPD-11147")) {
+			serviceContext.setAttribute(
+				"friendlyURLAssetCategoryIds",
+				serviceContext.getAssetCategoryIds());
+		}
```

Comment:
Q: The assignment of the asset category ids to "friendlyURLAssetCategoryIds" is a temporary artifact to manually test the feature, right? As defined in the story, the asset categories for the URL must be independent of the ones for the blogs entry.

If this is just here to provide a way for demoing the feature in its current state, I'm totally fine with it.

Reply:
I would think so, yes